### PR TITLE
Feature/block explorer link

### LINKED ISSUE #611: Add link to block explorer on the profile page
On both the private and public profile pages, we can add a "see on block explorer" to check user data in more depth

### PR DESCRIPTION
**Pull request by NeoVier**
## What issue does this PR close
Closes #611

## Changes Proposed ( a list of new changes introduced by this PR)
- Change the `Environment` type to be more aware of where we are: `Development` (localhost), `Staging`, `Demo` or `Production`. We can extract all that information just by looking at the current URL.
- Add a "See on block explorer" button on the profile page

## How to test ( a list of instructions on how to test this PR)
1. Go to someone's profile page
2. See if the "See on block explorer" button is there and if it redirects you to the correct page
3. Do the same for your own profile

### CODE REVIEW

File: tests/Session/SharedTests.elm [hunk 1]
```diff
@@ -183,3 +185,87 @@ communityDomainOnDevelopment =
...
+                \() ->
+                    makeUrl "verdes.cambiatus.io"
+                        |> Shared.environmentFromUrl
+                        |> Expect.equal Shared.Production
+            , fuzz (Fuzz.cambiatusUrl (Just ".cambiatus.io")) "correctly identifies prod communities" <|
+                \urlFuzz ->
+                    Shared.environmentFromUrl urlFuzz
+                        |> Expect.equal Shared.Production
+            ]
+        , describe "when on demo"
```
Comment (lucca65) [inline tests/Session/SharedTests.elm:224]: nice!

**Review by lucca65 — DISMISSED** [review]
Nice! do you think we can add a little something more? Actually navigating in the app I've remembered that a long time ago I had a dream of having cambiatus be super transparent with transaction ID on every action done on the blockchain... Maybe we can start by adding the transaction id on transfers? Do you think we can do it on this PR? otherwise I can open another issue...

here is the format for a transaction:

https://local.bloks.io/transaction/3e2abd9da7275e6c98c678c15c6807062a3d8a7f0b52d2ef6116d9b87824de50?nodeUrl=https%3A%2F%2Fstaging.cambiatus.io&systemDomain=eosio

this ID is found under the key `createdTx` on the `transfer` type

**Comment by NeoVier** [comment]
Definitely! I love this idea, and I think it's fitting to do it in this PR, as we can have some better architecture around making these block explorer links. Also, should be quick and easy to do it :smile:

**Comment by lucca65** [comment]
damn bro, I think I'm gonna cry, Cambiatus is getting so beautiful!

**Comment by NeoVier** [comment]
@lucca65 since we're doing this, we might as well do it right and include the `coreSymbol` param as well!

Can you list out which ones are related to each environment?

**Comment by lucca65** [comment]
Okay!

Here is the list:

- Staging: SYS
- Demo: SYS
- Prod: EOS

**Review by lucca65 — APPROVED** [review]
working like a charm 🧿

**Pull request merged**